editor/components/LightProps/GroupPanel/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { Box, List, IconButton, Stack, Button } from "@mui/material";
+import { TabPanel } from "@mui/lab";
+
+import DeleteIcon from "@mui/icons-material/Delete";
+// import EditIcon from "@mui/icons-material/Edit";
+
+import LEDcontrols from "../LEDcontrols";
+import OFcontrols from "../OFcontrols";
+
+import {
+  ControlMapStatus,
+  LED,
+  Fiber,
+  PartType,
+  PartPayloadType,
+} from "core/models";
+import { removeGroup, setSelectedParts, setSelectionMode } from "core/actions";
+
+import { reactiveState } from "core/state";
+import { useReactiveVar } from "@apollo/client";
+
+import { PART } from "constants";
+
+const GroupPanel = ({
+  partType,
+  groupName,
+  parts,
+  currentDancers,
+  currentStatus,
+  colorMap,
+}: {
+  partType: PartType;
+  groupName: string;
+  parts: string[];
+  currentDancers: string[];
+  currentStatus: ControlMapStatus;
+  colorMap: { [key: string]: string };
+}) => {
+  const dancers = useReactiveVar(reactiveState.dancers);
+
+  parts = parts.sort();
```

Comment:
I think changing the props is quite weird since it's passed by reference.
Can you do a survey for why redefining the `parts` property here?

Reply:
done